src/components/app-progress-bar-line/AppProgressBarLine.styles.js

```diff
@@ -0,0 +1,35 @@
+import palette from '~/styles/app-theme/app.pallete'
+
+export const styles = {
+  wrapperProgress: {
+    width: '100%',
+    display: 'flex',
+    flexDirection: { xs: 'row-reverse', sm: 'column' },
+    alignItems: { xs: 'center' },
+    marginTop: { xs: '8px', sm: '40px' }
+  },
+  labels: {
+    width: { xs: 'auto', sm: '100%' },
+    display: 'flex',
+    justifyContent: 'space-between',
+    marginBottom: { xs: '0', sm: '10px' },
+    marginLeft: { xs: '20px', sm: '0' }
+  },
+  progress: {
+    display: 'flex',
+    alignItems: 'center',
+    width: '100%',
+    height: '10px',
+    background: palette.primary[100],
```

Comment:
background 'primary.100'
And will not be necessary to import palette

Reply:
Fixed